setup.py

```diff
@@ -82,4 +85,8 @@
             wagtail=wagtail.bin.wagtail:main
     """,
     zip_safe=False,
+    cmdclass={
+        'sdist': add_subcommand(sdist, [('assets', None)]),
```

Comment:
Just been speaking to @tomdyson and realised that ``setup.py install`` is run by pip. Obviously, we don't want pip to try calling npm so would need to take that into account.

Maybe one simpler option could be for setup.py install to check if the css has been built and if not, show a warning with useful instructions to the user? This would mean regular pip users wouldn't get bothered but someone downloading from git and running ``setup.py install`` would be told if they forgot to build css.